front-end/src/components/Reactionbar/ReactionButton.tsx

```diff
@@ -112,6 +113,13 @@ const ReactionButton = function ({
 			</Button>
 		</span>
 	);
+
+	return (userNames.length > 0 ?
+		<Popup
+			content={userNames.join(', ')}
```

Comment:
It's an edge case, but let's say we have 20 users or so reacting to something, should we do something like "Pierre, Anna, Sid, Marie, Hannes, Peter and 14 others liked this" rather than listing all 20 names?

Reply:
Hopefully not an edge case for too long!

Reply:
done, thanks for pointing out